docs/reference/release-notes/7.11.asciidoc

```diff
@@ -1,8 +1,82 @@
-[[release-notes-7.11.0]]
-== {es} version 7.11.0
+[[release-notes-7.11.1]]
+== {es} version 7.11.1
 
 Also see <<breaking-changes-7.11,Breaking changes in 7.11>>.
 
+[[enhancement-7.11.1]]
```

Comment:
I don't think we need to mention the LDAP issue as it's already covered in the 7.11.0 notes:
https://www.elastic.co/guide/en/elasticsearch/reference/7.x/release-notes-7.11.0.html#known-issues-7.11.0